docs/run_test/run.md

```diff
@@ -0,0 +1,24 @@
+# Run LISA
+
+- ## [Quick start](quick_run.md)
+  - ### [Getting started with Azure](quick_run.md)
+- ## [Test classification](#test-classification)
```

Comment:
We don't need to classify those test cases. Legacy tests are part of Microsoft tests. It can be a subcategory of Microsoft tests.